docs/changehistory/NextVersion.md

```diff
@@ -459,6 +459,10 @@ The [NodeKey]($presentation-common) object contains a `pathFromRoot` attribute w
 
 In `3.0` changes have been made that changed the way this attribute is calculated, which means the same node produced by pre-3.0 and 3.x versions of `imodeljs` will have keys with different `pathFromRoot` value. To help identify the version of `NodeKey` a new `version` attribute has been added, with `undefined` or `1` being assigned to keys produced by pre-3.0 and `2` being assigned to keys produced by `3.x` versions of imodeljs. In addition, a new [NodeKey.equals]($presentation-common) function has been added to help with the equality checking of node keys, taking their version into account.
 
+## `KeySetJSON` in `@itwin/presentation-common`
+
+The format of `KeySetJSON` has been changed to reduce its size. Instead of containing an array of instance IDs it now contains a single compressed IDs string. See `CompressedId64Set` for more details about compressing IDs.
```

Comment:
Turn `KeySetJSON` and `CompressedId64Set` into doc links.

Reply:
There are several problems with doc links in changelogs:
- When at some point the API is removed, old changelogs containing links to that API need to be updated (by changing them to `backticks`)
- The markdowns contain broken links if you look at them directly on github (e.g. try clicking on one of the links in [this changelog](https://github.com/imodeljs/imodeljs/blob/master/docs/changehistory/2.9.0.md)). They only work on our doc site.

However, I do like the ability to click on APIs and get directed to reference docs... So I guess a decision has to be made whether the above problems can be ignored in favor of having working links on doc site.

Reply:
I think telling someone to "see XYZ for more info" and forcing them to figure out on their own how to find "XYZ" is bad. Having to fix broken links in ancient change logs is annoying; maybe @williamkbentley knows if that could be avoided somehow.

Reply:
I agree with @pmconne, `NextVersion.md` should use links to documentation. Instead, we should avoid links in `rush change` logs because these entries will not always be read on our documentation site.